en/Extensions/Community plugins.md

```diff
@@ -0,0 +1,47 @@
+Learn how to extend Obsidian with plugins built by the community. Use plugins to adapt Obsidian to fit your specific needs, such as support for additional file formats or integrating with a third-party service.
+
+> [!warning]
+> Community plugins run third-party code on your behalf that could potentially do harm. To learn more about what the Obsidian team does to prevent harmful plugins, refer to [[Plugin security]].
+
+## Install a community plugin
+
+To install a community plugin, you first need to turn off [[#Restricted mode]].
+
+1. Open **Settings**.
+1. In the side menu, select **Community plugins**.
+1. Select **Browse** to explore available community plugins.
+1. Select the plugin you want to install.
+1. Select **Install**.
+
+To use an installed plugin, you need to enable it.
+
+## Enable a community plugin
+
+Either select **Enable** directly after installing it, or enable it from the list of Community plugins under **Settings > Community plugins > Installed plugins**.
+
+## Update plugins
+
+Community plugins don't update automatically. You can either update all plugins at once, or update individual plugins.
```

Comment:
I think it helps to state that for security concerns, we don't update plugin automatically. Just so people don't think we don't do it to be a PITA 🤣

Reply:
I'll add it!